### ISSUE TITLE
Bug: Custom Token uses incorrectly scoped permission for users

### ISSUE THREAD

**Issue by genehynson** [influxdata/ui#3849]
There is a discrepancy between how we are creating All Access Tokens and Custom Tokens in regards to the `users` permission.

All Access Token:

![image](https://user-images.githubusercontent.com/6411855/153909624-707fb135-40d2-4b12-b251-0b51d5519b5c.png)

Custom Token with all boxes checked:

![image](https://user-images.githubusercontent.com/6411855/153909653-3264bd9d-f8db-4737-8c36-c8400de12280.png)

Notice how the `user` permission for the All Access token contains the user ID (aka `id`) but the Custom Token contains the orgID.  This is a very important difference. From @wbaker85:

> the custom one gets you read/write:org/<org_id>/users yeah and the all-access one gives you read/write:users/<user_id>. the problem is that creating an authorization very explicitly requires one and not the other. specifically, the requesting token must have precisely write:users/user_id for it to work

Basically if I create a Custom Token, I _cannot_ use that token to create other tokens. I don't think this is an intentional difference between these token generation methods and we should resolve it to avoid confusion from users who expect to be able to use their Custom Tokens to create other tokens.
